### PR TITLE
[plugin.video.tv3.cat] 1.3.9

### PR DESCRIPTION
**Pull request by jqandreu**
### Description

Added german translation and changed playback videos to HD
### Checklist:

- [ x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [ x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by enen92** [comment]
It would be nice if you could migrate the strings.xml to the new gettext format. See:

https://kodi.wiki/view/Language_support

**Comment by jqandreu** [comment]
Hi enen92

I migrated strings.xml files to strings.po.  One test have failed now. I dont understand what has failed

File: plugin.video.tv3.cat/resources/language/resource.language.ca_es/strings.po [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# Kodi Media Center language file
```
Comment (enen92) [inline plugin.video.tv3.cat/resources/language/resource.language.ca_es/strings.po]: Language folder layout is not compatible with Gotham but only Isengard and onwards. Please keep the old folder structure but use the new .po files

Comment (jqandreu) [reply plugin.video.tv3.cat/resources/language/resource.language.ca_es/strings.po]: Changed to old folder structure with new .po files. Thanks

**Pull request merged**